### PR TITLE
Local storage for wal-g

### PR DESCRIPTION
**Pull request by iamrz1**
xref: kubedb/project#475

### CODE REVIEW

**Comment by iamrz1** [comment]
fixes: kubedb/project#475

**Comment by iamrz1** [comment]
Added missing prefixes, fixes: https://github.com/kubedb/project/issues/486

**Comment by iamrz1** [comment]
- [x] Code

- [ ] Docs

File: test/e2e/postgres_test.go [hunk 1]
```diff
@@ -1242,6 +1242,69 @@ var _ = Describe("Postgres", func() {
 				f.EventuallyCountTable(postgres.ObjectMeta, dbName, dbUser).Should(Equal(6))
 			}
 
+			archiveAndInitializeFromLocalArchive := func() {
```
Comment (the-redback) [inline test/e2e/postgres_test.go]: `archiveAndInitializeFromArchive` from line https://github.com/kubedb/postgres/pull/270/files#diff-f6266047a19fe0b8a3eef454f2775e63R1180 is similar method. Don't need another method, use `archiveAndInitializeFromArchive` instead.

Just skip `err := f.CreateSecret(secret)` when file system is used.

File: test/e2e/postgres_test.go [hunk 2]
```diff
@@ -1276,6 +1339,106 @@ var _ = Describe("Postgres", func() {
...
+									},
+								},
+							},
+						}
+
+					})
+					Context("Archive and Initialize from wal archive", func() {
+						It("should archive and should resume from archive successfully", archiveAndInitializeFromLocalArchive)
+					})
+					Context("WipeOut wal data", func() {
```
Comment (the-redback) [inline test/e2e/postgres_test.go]: Remove wipeOut local wal data as the local dataq is not wipedOut

**Pull request merged**